libvast/src/format/pcap.cpp

```diff
@@ -109,22 +122,23 @@ const char* reader::name() const {
 }
 
 vast::system::report reader::status() const {
+  VAST_ASSERT(last_stats_);
```

Comment:
Do we have `VAST_ASSERT_MSG` or something like that so we can provide a useful assert message for this precondition?

Reply:
We can always do `last_stats_ && "message"`, but seems to be unnecessary here. The assertion is reported with file and line.

Reply:
I think it's still valuable to throw the user a message in the face. It helps us developers to quickly see what invariant got violated, instead having to start digging why, e.g., a to-bool-convertible variable was false.

Reply:
Great, thanks.